action.yml

```diff
@@ -11,7 +11,7 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - run: GOBIN="${{ github.action_path }}/bin/" go install github.com/pulumi/go-licenses@ec48b64
+    - run: GOBIN="${{ github.action_path }}/bin/" go install github.com/pulumi/go-licenses/v2@latest
```

Comment:
nit. we usually try to pin dependencies in our actions. (probably not a big deal here though since this is the sole user of our internal fork